claim-db-worker/components/prismaSchemaEditor/BusyPanel.tsx

```
@@ -0,0 +1,20 @@
+"use client";
+
+import React from "react";
+
+export default function BusyPanel({ isPulling }: { isPulling: boolean }) {
+  return (
+    <div className="flex-1 p-1 bg-[#181b21] flex flex-col rounded-lg">
+      <div className="flex-1 flex items-center justify-center">
+        <div className="text-center">
+          <svg className="h-8 w-8 animate-spin mx-auto mb-3" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)